versions/releases.py

```diff
@@ -163,7 +166,31 @@ def get_artifactory_download_data(url):
     }
 
 
-def get_release_notes_for_version(version_pk):
+def get_release_notes_for_version_s3(version_pk):
+    """Retrieve the adoc release notes from S3 and return the converted html string
+    """
+    try:
+        version = Version.objects.get(pk=version_pk)
+    except Version.DoesNotExist:
+        logger.info(
+            "get_release_notes_for_version_s3_error_version_not_found",
+            version_pk=version_pk,
+        )
+        raise Version.DoesNotExist
+    # get_content_from_s3 only works for keys with matching keys
+    # in the STATIC_CONTENT_MAPPING. Use get_file_data directly instead.
+    response = get_file_data(
+        get_s3_client(),
+        settings.STATIC_CONTENT_BUCKET_NAME,
+        f"release-notes/master/{version.name}.adoc",
```

Comment:
I'm not sure what the exact naming convention of the files will be.  This may not be correct.

Reply:
Based on what I'm seeing in https://github.com/boostorg/website/tree/master/feed/history, it looks like the naming convention is `version.slug.replace("-", "_")`.